include/z64math.h

```diff
@@ -1,11 +1,8 @@
 #ifndef Z64MATH_H
 #define Z64MATH_H
 
+#include "libc/math.h"
```

Comment:
```suggestion
#include "math.h"
```

From what I was told, libc headers should not have explicit folder paths to make them look like libc headers. Personally I disagree with this, I think the explicit header path makes everything simpler, it makes it easy to write a regex if you want to replace all decomp libc headers with up-to-date libc headers (e.g. `#include <math.h>`), makes it easy to organize the headers.

But this is what I've been told is what you're suppose to do.

Reply:
I did this preemptively, because soon I plan to remove the "z64" prefix from headers. And ideally, this game-side header would be called math.h. So adding that path would make things more clear.

But, I guess regardless, if libc is in the include path, then there will be clashes. Not sure how to resolve but I'll figure it out when we get there.